src/.nuget/dir.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+
+  <Target Name="VerifyVCRedist" BeforeTargets="GetSymbolPackageFiles" Condition="'$(_runtimeOSFamily)' == 'win'">
```

Comment:
Who publishes that package/where do they get the binaries from? If we did the same here, wouldn't that have the same issue, since whoever publishes the VC Redist bits would have to get them from program files anyways?